libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/CaliforniaId.tsx

```diff
@@ -1,7 +1,11 @@
 import { Regex } from '@monorepo/expo/shared/static';
-import { CardWrapper, Input } from '@monorepo/expo/shared/ui-components';
+import {
```

Comment:
Just reviewed, same feedback as @shiozakiyuko

Reply:
Hey @Davit-BetterAngels 

It's getting closer!
The last words should be "same CA ID", not "same name".

I confirmed the Submit button looked good. Thanks!

<img src="https://github.com/user-attachments/assets/2c6224c4-316c-430b-9c50-8a02a1186d0f" width="300"  />

Reply:
One more thing, @Davit-BetterAngels 

I just noticed now that when I tapped the X button instead of No to close the dialog, the error message didn't show up. I think it should behave as same as No. 

Thanks!

Reply:
Sorry @Davit-BetterAngels 

Everything looks good on iOS.

One more thing on Android that I noticed. I didn't notice this earlier though...
Somehow the background of the modal is shifted. Please check the attachment.

<img src="https://github.com/user-attachments/assets/b41a2309-211e-4c28-aca3-e74808056f0f" width="300" />

Reply:
we have a task to change modal library that might fix it. if not we can come back to this

Reply:
Summary of the issue:

Triggering a modal from a form field (while keyboard is open) does render the modal on full-screen.
- modal does render, but does not extend over the area previously covered by the keyboard.
- issue detected on certain Android devices (not all).
- issue no detected on iOS, though it may be possible it happens on older devices or OS's.